lua/nvim-lightbulb/init.lua

```diff
@@ -259,7 +259,7 @@ NvimLightbulb.update_lightbulb = function(config)
     pcall(vim.b[bufnr].lightbulb_lsp_cancel)
     vim.b[bufnr].lightbulb_lsp_cancel = nil
   end
-  local context = { diagnostics = vim.lsp.diagnostic.get_line_diagnostics() }
+  local context = { diagnostics = vim.diagnostic.get() }
```

Comment:
```suggestion
  local context = { diagnostics = vim.diagnostic.get(0, { lnum = vim.fn.getcurpos()[2] }) }
```
Or something like that. `vim.diagnostic.get()` with no args [returns all diagnostics for all buffers](https://neovim.io/doc/user/diagnostic.html#vim.diagnostic.get()). Same below.

Reply:
See comment below.